src/controllers/dynakube/istio/serviceentry.go

```diff
@@ -71,20 +71,24 @@ func buildServiceEntryIP(meta metav1.ObjectMeta, host string, port uint32) *isti
 
 func handleIstioConfigurationForServiceEntry(istioConfig *configuration) (bool, error) {
 	probe, err := kubeobjects.KubernetesObjectProbe(ServiceEntryGVK, istioConfig.instance.GetNamespace(), istioConfig.name, istioConfig.reconciler.config)
-	if probe == kubeobjects.ProbeObjectFound {
+	switch probe {
+	case kubeobjects.ProbeObjectFound:
 		return false, nil
-	} else if probe == kubeobjects.ProbeUnknown {
-		log.Error(err, "istio: failed to query ServiceEntry")
+	case kubeobjects.ProbeUnknown:
+		log.Error(err, "failed to query ServiceEntry")
 		return false, err
+	case kubeobjects.ProbeTypeNotFound:
+		log.Info("ServiceEntry type not found, skipping creation")
+		return false, nil
```

Comment:
this logic is the same (except the log) for the VirtualService and ServiceEntry logic.

wouldn't it be nicer to wrap the `kubeobjects.KubernetesObjectProbe` call with this logic, instead of calling it twice